script.js

```diff
@@ -0,0 +1,88 @@
+const API_KEY = "h9sWQ7IaXwu93R1HW7b6NydfTSn1da7x";
+
+const URL = "http://dataservice.accuweather.com/forecasts/v1/daily/5day/";
+
+const wrapper = document.querySelector(".wrapper");
+const input = document.querySelector("#input1");
+const btnWeather = document.querySelector("#btnWeather");
+const wrapperResult = document.querySelector("#result");
+
+const xhr = new XMLHttpRequest();
+
+const locationCity = {
+  Lviv: 324561,
+  Kyiv: 324505,
+  Kharkiv: 323903,
+  Chernihiv: 322162,
+  Odesa: 325343,
+  Mariupol: 323037,
+  Poltava: 325523,
+  Zhytomyr: 326609,
+  Cherkasy: 321985,
+  London: 328328,
+  Warsaw: 274663,
+  "New York": 349727,
+};
+
+const getLocationCity = function () {
+  for (const city in locationCity) {
+    if (input.value === city) {
+      return locationCity[city];
+    }
+  }
+  const p = document.createElement("p");
+  p.innerHTML = "You entered wrong city.";
+  wrapper.append(p);
+};
+
+const getInfoWeather = function (resultDayly) {
+  for (const infoWeather of resultDayly) {
+    const day = new Date(infoWeather.Date).getDate();
+    const month = new Date(infoWeather.Date).getMonth();
+    const year = new Date(infoWeather.Date).getFullYear();
+
+    const minTemperature = Math.round(
+      (infoWeather.Temperature.Minimum.Value - 32) / 1.8
+    );
+
+    const maxTemperature = Math.round(
+      (infoWeather.Temperature.Maximum.Value - 32) / 1.8
+    );
+
+    const weatherDay = infoWeather.Day.IconPhrase;
+    const weatherNight = infoWeather.Night.IconPhrase;
+
+    const wrapperWeather = document.createElement("div");
+    wrapperWeather.classList.add("wrapperWeather");
+    wrapperResult.append(wrapperWeather);
+
+    const resultWeather = `
+        <p class='date'>${day}/${month}/${year}</p>
+        <p class='temperature'>Min:<strong>${minTemperature}°C</strong> Max:<strong>${maxTemperature}°C</strong></p>
+        <p class='dayNight'>Day: ${weatherDay}</p>
+        <p class='dayNight'>Night:${weatherNight}</p>`;
+    wrapperWeather.innerHTML = resultWeather;
```

Comment:
It can be simplified 
```suggestion
 wrapperWeather.innerHTML = `
        <p class='date'>${day}/${month}/${year}</p>
        <p class='temperature'>Min:<strong>${minTemperature}°C</strong> Max:<strong>${maxTemperature}°C</strong></p>
        <p class='dayNight'>Day: ${weatherDay}</p>
        <p class='dayNight'>Night:${weatherNight}</p>`;
```